common/lib/dependabot/shared_helpers.rb

```diff
@@ -262,17 +265,18 @@ def self.reset_global_git_config(backup_path)
       FileUtils.mv(backup_path, GIT_CONFIG_GLOBAL_PATH)
     end
 
-    def self.run_shell_command(command)
+    def self.run_shell_command(command, escape_command_str: true)
       start = Time.now
-      stdout, process = Open3.capture2e(command)
+      cmd = escape_command_str ? escape_command(command) : command
```

Comment:
As mentioned above, my brain pulled the handbrake on this conditional as escaping optionally makes me nervous.